ITC205_BaseLine_Code/src/Main.java

```diff
@@ -1,5 +1,7 @@
 import java.text.SimpleDateFormat;
 import java.util.Scanner;
+//Ready for static review
```

Comment:
D23 - Main  line 28    change library.getInstance()  to  Library.getInstance()

Reply:
I have also amended line 14 library.getInstance() --- Library.getInstance - in **ReturnBookControl** class to align with D23

Reply:
Confirmed Fixed - Facilitator